### PR TITLE
Added homework for JS2 week1

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

File: javascript/javascript2/week1/homework app2/main.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+let ul = document.querySelector(".listOfProducts");
+
+function renderProducts(ourProducts) {
+    for (let i = 0; i < ourProducts.length; i++) {
+        const productNames = ourProducts[i];
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/homework app2/main.js]: just a minor note: I would favor using singular names for single values and in this case is also not really a productName you are getting but  a "product" itself 🐱‍👓 if someone(also future you) was reading and using your code afterwards it wouldn't be that clear 😇

File: javascript/javascript2/week1/homework app2/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        const productTitle = document.createElement("h3");
+        const productPrice = document.createElement("p");
+        const productRating = document.createElement("p");
+        productTitle.innerText = productNames.name;
+        productPrice.innerText = "Price :" + productNames.price;
+        productRating.innerText = "Rating :";
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/homework app2/main.js]: in here you have an extra ";" probably a typo as you are missing a "+", that prevents the ratings from showing

File: javascript/javascript2/week1/homework app2/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+        let list = document.createElement("li");
+        list.appendChild(productTitle);
+        list.appendChild(productPrice);
+        list.appendChild(productRating);
+        ul.appendChild(list);
+    }
+}
+
+renderProducts(products);
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/homework app2/main.js:26]: Well done here ! 🤠

File: javascript/javascript2/week1/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    for (let i = 0; i < danishWords.length; i++) {
+        if (danishWords[i].length < danishWords[shortest].length) {
+            shortest = i;
+        }
+    }
+    return danishWords[shortest];
+}
+
+shortest = getShortestWord();
+console.log(shortest);
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/index.js:16]: Very cool and neat solution ! 👸

File: javascript/javascript2/week1/index.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+const uniqueLetter = ["å", "æ", "ø"];
+
+function countLetters(str) {
+    let result = { total: 0 };
+
+    for (let i = 0; i < str.length; i++) {
+        if (uniqueLetter.includes(str[i])) {
+            result.total = result.total + 1;
+            if (result.hasOwnProperty(str[i])) {
+                result[str[i]] = result[str[i]] + 1;
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/index.js]: in here one might prefer using += to increase and update the value 

 `result[str[i]] += 1;`

File: javascript/javascript2/week1/index.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+button.addEventListener("click", function() {
+    let random = spiritAnimals[Math.floor(Math.random() * spiritAnimals.length)];
+    const name = document.querySelector(".nameInput").value;
+    if (name === "") {
+        alert("Please enter your name to get spirit animal name");
+    } else {
+        newP.innerText = "Your spirit animal name: " + name + "-" + random;
+        body.appendChild(newP);
+    }
+});
```
Comment (Falafelqueen) [inline javascript/javascript2/week1/index.js:71]: Works nicely 🧚‍♀️

**Review by Falafelqueen — APPROVED** [review]
Great job! Just fix the typo and you are ready to shine🦸‍♀️

File: javascript/javascript2/week1/homework app2/main.js (hunk 1 above)
Comment (SvitlanaBurlaka) [reply javascript/javascript2/week1/homework app2/main.js]: Thank you kindly for your review. Your comments make sense and i fixed all what you said.  I'll pay attention more for naming elements next time!

**Pull request merged**